hw02/hw02_exploring_gapminder.Rmd

```diff
@@ -0,0 +1,65 @@
+library(gapminder)
+library(ggplot2)
+library(palmerpenguins)
+library(tidyverse)
+library(dplyr)
+
+# 1.1 & 1.2
+
+gapminder %>%
+  select(country, year, gdpPercap) %>%
+  filter(country %in% c("Peru" , "Japan", "Canada"), year>1969 & year<1980)
+
+# 1.3
+
+gapminder %>%
+  mutate(lifeExpDrop = lag(lifeExp)
+
+# 1.4 
+
+gapminder %>%
+  select(country, gdpPercap) %>%
+  mutate(maxgdpPercap = max(gdpPercap)) %>%
+  group_by(country)
+
+# 1.5
+
+gapminder %>%
+  filter(country == "Canada") %>%
+  select(country, lifeExp, gdpPercap) %>%
+  ggplot() + aes(x=lifeExp, y=gdpPercap) + geom_point() +
+  scale_y_continuous(trans = "log10")
```

Comment:
Nice job! It may be helpful to add axis titles in the future but overall your scatterplot looks great!